### ISSUE TITLE
Manual for TransactionScope is incorrect

### ISSUE THREAD

**Issue by karlra** [MicrosoftDocs/feedback#2192]
**Link to article:**
https://docs.microsoft.com/en-us/dotnet/api/system.transactions.transactionscope?view=netcore-3.0

**Problem:**
In the code example, this section:

```
// The Complete method commits the transaction. If an exception has been thrown,
// Complete is not  called and the transaction is rolled back.
scope.Complete();
```

appears to be incorrect. Comparing against the implementation at https://referencesource.microsoft.com/#system.transactions/System/Transactions/TransactionScope.cs,f78c5d9afd50f246, the transaction is not committed until the TransactionScope is disposed, which is extremely unintuitive and potentially dangerous to misunderstand. Calling .Complete() only appears to set the completed bool.

**Comment by mlottner** [comment]
@karlra Thanks for taking the time to give us feedback. 
I've reassigned your issue to the .NET team for awareness and potential fix/update. 
Appreciate your contribution!

**Comment by mimisasouvanh** [comment]
@scottaddie Can you take a look at this?

**Comment by scottaddie** [comment]
@ajcvickers @bricelam Can one of you please look at this?

**Comment by ajcvickers** [comment]
@scottaddie Yeah, looks like the documentation is incorrect. However, this isn't an EF or ADO.NET type--it will need to be assigned to whomever owns System.Transactions in CoreFx.
